site/docs/v1/tech/email-templates/index.adoc

```diff
@@ -36,6 +36,27 @@ Here are the topics in this section:
 
 * link:/docs/v1/tech/email-templates/message-templates/[Message Templates]
 
+== Common Providers
+
+=== SendGrid
+
+[NOTE]
+====
+This information is for starting guidance only - FusionAuth connects seamlessly with many different email services. A deep exploration of email best practices and providers is outside the scope of this section.
+====
+
+SendGrid is a commonly used email provider and integrates well with FusionAuth via API. To setup SendGrid, please follow the instructions below:
+
+- https://docs.sendgrid.com/for-developers/sending-email/integrating-with-the-smtp-api
+
+Other resources related to SendGrid are linked below:
+
+1. https://docs.sendgrid.com/for-developers/sending-email/getting-started-with-transactional-emails
+1. https://docs.sendgrid.com/for-developers/sending-email/sender-identity
+1. https://docs.sendgrid.com/ui/account-and-settings/how-to-set-up-domain-authentication
+
+Using dedicated IP addresses for any SendGrid account will build email reputation over time (improving email deliverability). And lastly, to control spoofing, phishing, and spam, it is recommended to set up SPF and DKIM for the primary email domain (such as `example-company.com`). Emails should also be sent from an address such as `no-reply@example-company.com`.
```

Comment:
I don't know if there is a standard way of talking about SPF and DKIM, but these are essentially both DNS records. So sometimes we talk about it as an "SPF record" or a "DKIM record". These are sort of advanced topics, so if we can find a good link that describe this that may be helpful. Perhaps SendGrid has some good descriptions of how and why to configure these two records.

Reply:
Added some links!